docs/source/changes.rst

```diff
@@ -1,20 +1,48 @@
 Changelog
 =========
 
-v2.0 (Development)
-------------------
+v2.0 (November 1, 2018)
+-----------------------
 
-- Python 2 is no longer supported
+PyQuil 2.0 is a major release of pyQuil: Rigetti's toolkit for constructing and running quantum
```

Comment:
I think a comma is more appropriate than a colon here.

Reply:
I also think we could just get rid of the post-colon language -- its directly copied from the release below

Reply:
consistency!